kent2cky/GeoSearch
# Release 1.0

**What does the PR do?**

Apply changes made in the release 1.0 branch to Develop 

**Description of tasks to be completed**

- Bring build files to root folder so the page can be hosted on github pages
- Organize files in the dist folder 

**How should this be manually tested?**

- After cloning the repo, `cd` into it run `npm install`
- Check that `js/main.min.js` exists on the root folder
- Check that js/jquery/jquery-3.4.1.min.js exists on the root folder
- Check that `css/main.css` exists on the root folder
- Check that js/jquery/jquery-3.4.1.min.js exists on the dist folder
- Check that `js/main.min.js` exists on the dist folder
- Check that css/main.css exists on the root folder
- Run the command `gulp` on the terminal and after the page loads check that everything works properly

**Any background context you want to provide?**

nil

**What are the relevant pivotal tracker stories?**

#3
